libraries/rib-base/src/main/java/com/badoo/ribs/android/integrationpoint/ActivityIntegrationPoint.kt

```diff
@@ -3,45 +3,47 @@ package com.badoo.ribs.android.integrationpoint
 import android.content.Intent
 import android.os.Bundle
 import android.view.ViewGroup
+import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.MutableLiveData
 import com.badoo.ribs.android.activitystarter.ActivityBoundary
 import com.badoo.ribs.android.activitystarter.ActivityStarter
+import com.badoo.ribs.android.dialog.DefaultDialogLauncher
+import com.badoo.ribs.android.dialog.DialogLauncher
 import com.badoo.ribs.android.permissionrequester.PermissionRequestBoundary
 import com.badoo.ribs.android.permissionrequester.PermissionRequester
+import com.badoo.ribs.android.store.RetainedInstanceStoreViewModel
 
-class ActivityIntegrationPoint(
+open class ActivityIntegrationPoint(
```

Comment:
From my point of view create own class of ActivityIntegreationPoint only for overriding DialogLauncher sounds strange, can we add DialogLauncher as constructor field?

Reply:
There is no meaning in exposing only `DialogLauncher` via contractor parameter. But at same time we can't expose `ActivityStarter` and `PermissionRequester` as default parameters because they depends on internal `RequestCodeRegistry`.

Please extend `ActivityIntegrationPoint` and override the field, use `createIntegrationPoint` to replace default one in activity or fragment.